src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx

```diff
@@ -160,31 +159,53 @@ describe("toggleButton for dictionary release status", () => {
         expect(getByTestId('2').closest('span')).not.toHaveClass('Mui-checked');
         expect(spyOnEditDictionaryVersion).not.toBeCalled();
     });
+});
 
-    describe('when user is not the owner of sources', () => {
-        it('should not open confirmation dialog onclick of dictionary toggle button', () => {
-            const {container, getByRole, getByTestId} = renderUI({
-                versions: [unreleasedVersion],
-                showCreateVersionButton: false
-            });
-            getByRole('checkbox').click();
-            const dialog = container.querySelector('[data-testid="confirm-dialog"]');
-            expect(dialog).toBeNull();
-            expect(getByTestId('2').closest('span')).not.toHaveClass('Mui-checked');
+describe('when user is not the owner of sources', () => {
+    it('should not open confirmation dialog onclick of dictionary toggle button', () => {
+        const {container, getByRole, getByTestId} = renderUI({
+            versions: [unreleasedVersion],
+            showCreateVersionButton: false
         });
+        getByRole('checkbox').click();
+        const dialog = container.querySelector('[data-testid="confirm-dialog"]');
+        expect(dialog).toBeNull();
+        expect(getByTestId('2').closest('span')).not.toHaveClass('Mui-checked');
+    });
 
-        it("should show tooltip with valid message on hovering the toggle button", async () => {
-            const {getByTitle, getByTestId} = renderUI({
-                versions: [unreleasedVersion],
-                showCreateVersionButton: false
-            });
-            const toggleButton: HTMLElement | null = getByTestId('2').closest('span');
-            expect(toggleButton).not.toBeNull();
-            toggleButton !== null && fireEvent.mouseMove(toggleButton);
-            expect(getByTitle("You don’t have permission to change the status")).toBeInTheDocument();
+    it("should show tooltip with valid message on hovering the toggle button", async () => {
+        const {getByTestId} = renderUI({
+            versions: [unreleasedVersion],
+            showCreateVersionButton: false
         });
-    })
+        const toggleButton: HTMLElement | null = getByTestId('2').closest('span');
+        expect(toggleButton).not.toBeNull();
+        toggleButton !== null && fireEvent.mouseMove(toggleButton);
+        console.log(getByTestId('2'));
```

Comment:
@tarunkumar-tw can you remove the console.log here

Reply:
@binduak removed in the next commit